rosdep/python.yaml

```
@@ -7963,6 +7963,13 @@ python3-nose-yanc:
   debian: [python3-nose-yanc]
   openembedded: [python3-nose-yanc@meta-ros-common]
   ubuntu: [python3-nose-yanc]
+python3-ntplib:
+  debian: [python3-ntplib]
+  fedora: [python3-ntplib]
+  nixos: [python3Packages.ntplib]
+  opensuse: [python3-ntplib]
+  rhel: ['python%{python3_pkgversion}-ntplib']
```

CI is failing here because apparently RHEL-7 doesn't have this.  This should work:

```suggestion
  rhel:
    '*': ['python%{python3_pkgversion}-ntplib']
    '7': null
```